src/index.ts

```diff
@@ -100,165 +23,45 @@ export = class OutputDcatAp201 {
     res.set('Content-Type', 'application/json');
 
     try {
-      const { domainRecord, siteModel } = await this.fetchDomainAndSite(req.hostname);
 
-      // Use dcatConfig query param if provided, else default to site's config
-      let dcatConfig = typeof req.query.dcatConfig === 'string'
-        ? this.parseProvidedDcatConfig(req.query.dcatConfig as string)
-        : req.query.dcatConfig;
-    
-      if (!dcatConfig) {
-        dcatConfig = _.get(siteModel, 'data.feeds.dcatAP201');
+      const { res: { locals: { feedTemplate } }, app: { locals: { feedTemplateTransformsDcatAp } } }: {
+        res?: {
+          locals?: {
+            feedTemplate?: any
+          }
+        }
+        app: {
+          locals: {
+            feedTemplateTransformsDcatAp?: TransformsList
+          }
+        }
+      } = req;
```

Comment:
This is a little confusing to read. could you instead just do:
```ts
const feedTemplate = req.res?.locals?.feedTemplate as any;
const feedTemplateTransformsDcatAp = req.app.locals.feedTemplateTransformsDcatAp as TransformsList;
```

Reply:
I agree. The destructor is quite complicated for only two variable. I have implemented the one you suggested.